features/step_definitions/ability_steps.rb

```diff
@@ -0,0 +1,69 @@
+Given("I have created several character abilities") do
+  FactoryBot.create(:character,
+                    :user => @registered_user,
+                    :name => "Trol",
+                    :description => "My first trol")
+
+end
```

Comment:
Step title says that you create character abilities, but you create only one character.